apps/common/lib/utils.ex

```diff
@@ -0,0 +1,14 @@
+defmodule Common.Utils do
+  @moduledoc "Module for utilities used across belethor."
+
+  @doc """
+  macro to correctly log debug messages.
+
+  Calls `Logger.debug` in a way that makes dialyzer happy and does impact performance in production.
+  """
+  defmacro debug(msg) do
```

Comment:
can we get analogues for the other log levels?

Reply:
yes